app/views/layouts/_nav.haml

```diff
@@ -17,8 +16,6 @@
               rotation_generator_roster_assignments_path(@roster)
       %ul.navbar-nav
         - if @current_user.present?
-          %li.nav-item.navbar-text.mx-2
-            Logged in as #{@current_user.full_name}
-          = nav_link_item 'Edit Profile',
+          = nav_link_item @current_user.full_name,
```

Comment:
Bootstrap 5 changed uses your browsers font by default now. I noticed the nav-bar got pretty squished because of that. I feel it's a pretty normal behavior to expect your name in a nav-bar to bring you to your profile, so I opted to make that change.